tests/dune

```diff
@@ -11,15 +11,3 @@
     main.output
     (run ./main.exe))
    (diff? main.expected main.output))))
-
-(executables
- (names solveEmpty standalone_minimal_maximization standalone_minimal)
- (modules
-  solveEmpty
-  standalone_minimal_maximization
-  standalone_minimal
-  rat
-  var
-  ex
-  instantiateBasicFunctor)
- (libraries ocplib-simplex))
```

Comment:
Why did you remove these files? They are useful for testing, they allow to quickly write simplex examples.

Reply:
Their content was moved into [tests/mail.ml](https://github.com/hra687261/ocplib-simplex/blob/1ef855e705584d0ec8625c21e2ba2779067c33b2/tests/main.ml). That way dune's testing system can be used, and it's easier to see differences with the previous testing results (through [tests/main.expected](https://github.com/hra687261/ocplib-simplex/blob/1ef855e705584d0ec8625c21e2ba2779067c33b2/tests/main.expected)), and to update the expected result with dune promote.

Reply:
The dune testing system was already working before, wasn't it ? Actually, I think the files you removes should be part of a library that would be used by `main` and other future tests. In the current state of the project, there is only one test started by dune, which is not enough ; if we want to add more non-regression tests, it would be simpler to already have the functor instanciated.

Reply:
What was being tested before the last commit was the tests that were in main.ml, it actually contained the three tests that were in the three files that were deleted, but rewritten as functions. I wrote them that way to have one executable instead of three, but I guess it's more readable if I separate them back into three files.
Currently, the functor is instantiated inside the simplex.ml file. And the three test files use that same functor to run the tests.
To add more tests, you'd need to add an executable, add the test's name to the `tests` stanza inside the dune file, and add the corresponding rule.